CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+3.x.x
+-----
+
+**CHANGES**
+- Remove support for Python 3.6 in aws-parallelcluster-batch-cli.
+- Upgrade Python version in API infrastructure, API Docker container and cluster Lambda resources.
```

Comment:
is it worth mentioning nodejs version upgrade?

Reply:
done